client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/extensions/Common.kt

```diff
@@ -0,0 +1,14 @@
+package ac.uk.ebi.biostd.client.extensions
+
+import org.springframework.http.HttpHeaders
+import org.springframework.util.LinkedMultiValueMap
+
+fun <K, V> LinkedMultiValueMap(vararg pairs: Pair<K, V>): LinkedMultiValueMap<K, V> {
```

Comment:
I wonder if we should use this also [here](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/common/BioWebClientRequestBuilder.kt#L17)